SingularityUI/app/components/deployDetail/ActiveTasksTable.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import Utils from '../../utils';
+
+import SimpleTable from '../common/SimpleTable';
+import JSONButton from '../common/JSONButton';
+
+const ActiveTasksTable = ({tasks}) => (
+  <SimpleTable
+    emptyMessage="No tasks"
+    entries={tasks}
+    perPage={5}
+    first={true}
+    last={true}
+    headers={['Name', 'Last State', 'Started', 'Updated', '', '']}
+    renderTableRow={(data, index) => {
+      return (
+        <tr key={index}>
+          <td><a href={`${config.appRoot}/task/${data.taskId.id}`}>{data.taskId.id}</a></td>
+          <td><span className={`label label-${Utils.getLabelClassFromTaskState(data.lastTaskState)}`}>{Utils.humanizeText(data.lastTaskState)}</span></td>
+          <td>{Utils.timestampFromNow(data.taskId.startedAt)}</td>
+          <td>{Utils.timestampFromNow(data.updatedAt)}</td>
+          <td className="actions-column"><a href={`${config.appRoot}/request/${data.taskId.requestId}/tail/${config.finishedTaskLogPath}?taskIds=${data.taskId.id}`} title="Log">&middot;&middot;&middot;</a></td>
+          <td className="actions-column"><JSONButton object={data}>{'{ }'}</JSONButton></td>
```

Comment:
more of a question than an actual issue / nitpick -- is a JSON button ever anything other than `{ }`? Would it be worth folding it into the element?